### DISCUSSION TITLE
Is it updated the API conformant product list?

### DISCUSSION THREAD

**Discussion by PabloSaro** [KhronosGroup/VulkanSC-Docs#10] [Q&A] [answered]
Hello Khronos group,
We are currently working on a project related to safety critical applications and we are talking to several vendors, one of them is offering us a platform (GPU) that supposedly has a driver that is vulkan sc compliant, however, this platform is not on the list.
Is it possible that the list is not up to date?
Greetings!!
Pablo

**Comment by dgkoch** [comment, marked as answer]
The conformant products list (https://www.khronos.org/conformance/adopters/conformant-products/vulkansc) is current with everything that has been submitted through the official Khronos conformance process. 

You'll have to ask the vendor to provide you with more details, but if they want to claim Vulkan SC conformance for a product on a "Target Implementation", it should either be listed as a conformant product directly, or they should be able to point to a "Reference Implementation" conformance submission for the Target implementation (See section P9 of the Conformance Process document at https://www.khronos.org/files/conformance_procedures.pdf for definitions of these terms).

**Reply by PabloSaro** [reply]
Perfect, thanks for the reply
